mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp

```diff
@@ -1874,6 +1900,15 @@ vectorizeUnPackOpPrecondition(tensor::UnPackOp unpackOp,
   return success();
 }
 
+/// Need to check if the inner-tiles are static/constant.
```

Comment:
Is it a TODO? Do you mean to move the checks from `vectorizeAsInsertSliceOp` to here?

Reply:
C&P Failure 🤦🏻 
* https://github.com/llvm/llvm-project/blob/main/mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp#L1855

:)